rust/src/cliwrap/kernel_install.rs

```diff
@@ -42,7 +45,12 @@ pub(crate) fn main(argv: &[&str]) -> Result<()> {
     }
     if let Some(k) = new_kernel {
         undo_systemctl_wrap()?;
-        run_dracut(&k)?;
+        let initoverlayfs_path = Path::new(INITOVERLAY_INSTALL_CMD);
+        if initoverlayfs_path.exists() {
+            run_initoverlayfs()?;
+        } else {
+            run_dracut(&k)?;
```

Comment:
You'll want to run initoverlayfs-install with all these args though:

```
        .args(&[
            "--no-hostonly",
            "--kver",
            kernel_dir,
            "--reproducible",
            "-v",
            "--add",
            "ostree",
            "-f",
        ])
```

so effectively dracut_path should be initoverlayfs-install in the initoverlayfs case... You could rename dracut_path to be something like initfs_builder...

Reply:
The intent is for "initoverlayfs-install" to be a drop in replacement for "dracut", with all the same args, kinda like "podman" is a drop in command for "docker"...